config/other-style.yml

```diff
@@ -383,9 +383,3 @@ Style/FrozenStringLiteralComment:
 
 Style/HashEachMethods:
   Enabled: true
-
-Style/HashTransformKeys:
```

Comment:
Ah, that's a shame. Can we set them to false (with a comment), instead, so we don't get the warning about not configuring them?

Reply:
Sure thing, I've added that.